pkgs/top-level/all-packages.nix

```diff
@@ -19224,6 +19224,8 @@ with pkgs;
 
   utf8proc = callPackage ../development/libraries/utf8proc { };
 
+  unicode-paracode = callPackage ../tools/misc/unicode { };
```

Comment:
I have used "unicode-paracode" here because with "unicode" I get
`error: value is a set while a Boolean was expected
`

Reply:
it is used as argument in different locations like `ncurses`.